docs/learn/learn-transaction-fees.md

```diff
@@ -24,23 +24,40 @@ designed the Polkadot fee system with the following objectives:
 
 Fees on the Polkadot Relay Chain are calculated based on three parameters:
 
-- A per-byte fee (also known as the "length fee").
-- A weight fee.
-- A tip (optional).
-
-The length fee is the product of a constant per-byte fee and the size of the transaction in bytes.
-
-Weights are a fixed number designed to manage the time it takes to validate a block. Each
-transaction has a base weight that accounts for the overhead of inclusion (e.g. signature
-verification) as well as a dispatch weight that accounts for the time to execute the transaction.
-The total weight is multiplied by a per-weight fee to calculate the transaction's weight fee.
-
-Tips are an optional transaction fee that users can add to give a transaction higher priority.
-
-Together, these three fees constitute the inclusion fee. This fee is deducted from the sender's
-account prior to transaction execution. A portion of the fee will go to the block producer and the
-remainder will go to the [Treasury](learn-treasury.md). At Polkadot's genesis, this is set to 20%
-and 80%, respectively.
+- A Weight fee
+  - Base weight
+  - Call(s) weight
+- A Length fee
+- A Tip (optional).
+
+Weights are a fixed set of numbers that are used in Substrate based chains to manage 
+the time it takes to validate a block. Each transaction has a base weight that accounts 
+for the overhead of inclusion (e.g. signature verification) and a dispatch weight that 
+accounts for the weight of a transaction (the time to execute the transaction). The weights 
+in the runtime are converted to a fee using 
+[WeightToFee](https://docs.substrate.io/how-to-guides/v3/weights/calculate-fees/).
+
+The weight fee is the sum of the base weight + sum of the total weight consumed by call(s).
+To learn more about the motivation of a weight fee, check out this 
+[Substrate doc](https://docs.substrate.io/v3/concepts/weight/) on weights.
+The base fee is the minimum a user pays for a transaction. 
+
+The length fee is a per-byte fee multiplier for the size of the transaction in bytes.
+
+There is also a targeted fee adjustment that serves as a multiplier which can tune the 
+final fee based on network congestion. This can constitute an adjusted weight fee which is 
+calculated as the targeted fee adjustment \* weight fee.
```

Comment:
Also just write it out. "times the weight fee"

Reply:
Updated